### PR TITLE
add sorted list of empty process resource specification to the

### PR DESCRIPTION
**Pull request by KochTobi**
## Add list of processes to base config 
Currently there are resource definitions for `cpu`, `mem` and `time` in some but not in all configuration profiles. 

I think it would be nice to have a sorted list of processes in the `base.config` file where each process defines its preferred resources. The maximal ressurces from the environment specific configuration profiles can then be used to check for limitation. 

The list should contain all processes. If I missed something feel free to add them.

Also there is no configuration for the single processes yet. It would be great if we could come up with some sort of preferred resource demands and add them together (over time?)

## PR checklist
 - [x] PR is made against `dev` branch
 - [ ] PR is a hotfix against `master` branch
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Ensure the test suite passes (`./scripts/test.sh -p docker -t ALL`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/SciLifeLab/Sarek/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by KochTobi** [comment]
I can't figure out what's wrong with my config.
`ERROR ~ Unknown config attribute process.withName:BuildWithDocker -- check config file: /home/travis/build/SciLifeLab/Sarek/nextflow.config
`

**Comment by MaxUlysse** [comment]
I'll check it out right now

**Comment by MaxUlysse** [comment]
I actually have no idea, but if you remove all of the process from `build.nf`, it seems to be working...

**Comment by KochTobi** [comment]
> I actually have no idea, but if you remove all of the process from `build.nf`, it seems to be working...

That' strange. Should I remove them then?

**Comment by MaxUlysse** [comment]
Yes, you definitively can, this `build.nf` script is not really part of the pipeline

File: conf/base.config [hunk 1]
```diff
@@ -47,11 +47,114 @@ process {
...
 
-  withName:ConcatVCF {
+  //## appearing in multiple .nf scripts
+  // - `RunBcftoolsStats` - Run BCFTools stats on vcf files
+  withName: RunBcftoolsStats {}
+  // - `RunVcftools` - Run VCFTools on vcf files
+  withName: RunVcftools {}
+  // - `CreateIntervalBeds` - Create and sort intervals into bed files
+  withName: CreateIntervalBeds {}
+  // - `ConcatVCF` - Merge results from paralellized callers
```
Comment (MaxUlysse) [inline conf/base.config]: parallelized

**Comment by MaxUlysse** [comment]
Closing, this this is handled differently since the port to nf-core

**Pull request closed without merging**